tests/test_data.py

```diff
@@ -0,0 +1,24 @@
+import app
+from app.assess.data import list_files_in_folder
+
+
+def test_list_files_in_folder(monkeypatch):
+    def mock_list_objects_v2(Bucket, Prefix):  # noqa
+        return {
+            "Contents": [
```

Comment:
Does this list need expanding? Doc and docx, etc

Reply:
The extension of the file isn't really relevant to the functionality, but it's easy enough to do so may aswell!

Reply:
@aaronwilliamsv1 thanks, I've expanded this in https://github.com/communitiesuk/funding-service-design-assessment/pull/235/commits/5ee36e4f9cb83c742f73cde58b43f01e51079f5a according to the [accepted files](https://github.com/communitiesuk/digital-form-builder/blob/b219aab2e9df3c8eda39947982bbaa3d4a226d1a/fsd_config/form_jsons/cof_r2/cy/cefnogaeth-leol.json#L66)